### ISSUE TITLE
[nifi-cluster] cluster.nifiProperties.overrideSecretConfig is ignored when configured

### ISSUE THREAD

**Issue by Nickmman** [konpyutaika/nifikop#462]
### What steps will reproduce the bug?

* Deploy helm chart with following values:
```
cluster:
  nifiProperties:
    overrideSecretConfig:
      name: nifi-config-override
      data: nifi.properties
      namespace: nifi
```
* Create a secret with the following contents:
```
apiVersion: v1
data:
  nifi.properties: bmlmaS5zZW5zaXRpdmUucHJvcHMua2V5PXpnOExlVlNZSkQ0cEhzR2pUd3Y5dXkK
kind: Secret
metadata:
  name: nifi-config-override
  namespace: nifi
type: Opaque
```
Decoded, the secret is as follows (in JSON):
```
{
  "nifi.properties": "nifi.sensitive.props.key=zg8LeVSYJD4pHsGjTwv9uy\n"
}
```
* Observe as the `nifi-cluster-config-1` secret does not contain the properties from the override secret

### What is the expected behavior?

The resulting node config contains the properties mixed in from the override secret

### What do you see instead?

The properties in the resulting node config remain unconfigured

### NiFiKop version

v1.10.0

### Golang version

Whatever is bundled with the container image

### Kubernetes version

Client Version: v1.29.1
Kustomize Version: v5.0.4-0.20230601165947-6ce0bf390ce3
Server Version: v1.29.7-gke.1104000

### Additional context

I have configured the values based on https://github.com/konpyutaika/nifikop/issues/47#issuecomment-1077539019.

I tried with both `nifi.properties` and `properties` as the secret key but neither worked.

Additionally, I tried with both the values for secret override above, as well as the following (per the comment reference to an old repo):
```
cluster:
  readOnlyConfig:
    nifiProperties:
      overrideSecretConfig:
        name: nifi-config-override
        data: nifi.properties
        namespace: nifi
```

**Comment by Nickmman** [comment]
@juldrixx Would you happen to have any insight on this issue?

**Comment by juldrixx** [comment]
> @juldrixx Would you happen to have any insight on this issue?

The feature was implemented in this PR https://github.com/konpyutaika/nifikop/pull/455, but has not yet been released. It is therefore not available in the current helm chart.

**Comment by Nickmman** [comment]
> > @juldrixx Would you happen to have any insight on this issue?
> 
> The feature was implemented in this PR #455, but has not yet been released. It is therefore not available in the current helm chart.

Ah, I see. Is there an ETA on when the next version will be released with this new feature?
